securityscanutils/issuewriter/issue_writer.go

```diff
@@ -0,0 +1,11 @@
+package issuewriter
+
+import (
+	"context"
+
+	"github.com/google/go-github/v32/github"
+)
+
+type IssueWriter interface {
```

Comment:
I like this interface, and think that it makes a lot of sense in the context of this PR. Since the idea is fresh, I know how this operates, but I'm sure in 1 month, or 6 months, I'll forget the contract of this Interface. Could you add a comment that clarifies how this is used?

I think also clarifying what each parameter of the Write function is, would be useful.